src/assets/wise5/authoringTool/node/advanced/node-advanced-authoring.component.html

```diff
@@ -3,25 +3,25 @@
     <md-icon>arrow_back</md-icon>
     <md-tooltip md-direction='top' class='projectButtonTooltip'>{{ ::'back' | translate }}</md-tooltip>
   </md-button>
-  <md-button ng-if='!$ctrl.isGroupNode($ctrl.node.id)'
+  <md-button ng-if='!$ctrl.node.isGroup()'
       class='topButton md-raised md-primary'
       ng-click='$ctrl.showGeneralAdvancedView()'>
     <md-icon>settings</md-icon>
     <md-tooltip md-direction='top' class='projectButtonTooltip'>{{ ::'generalAdvanced' | translate }}</md-tooltip>
   </md-button>
-  <md-button ng-if='!$ctrl.isGroupNode($ctrl.node.id)'
+  <md-button ng-if='!$ctrl.node.isGroup()'
       class='topButton md-raised md-primary'
       ng-click='$ctrl.showEditTransitionsView()'>
     <md-icon>trending_up</md-icon>
     <md-tooltip md-direction='top' class='projectButtonTooltip'>{{ ::'editTransitions' | translate }}</md-tooltip>
   </md-button>
-  <md-button ng-if='!$ctrl.isGroupNode($ctrl.node.id)'
+  <md-button ng-if='!$ctrl.node.isGroup()'
       class='topButton md-raised md-primary'
       ng-click='$ctrl.showEditConstraintsView()'>
     <md-icon>block</md-icon>
     <md-tooltip md-direction='top' class='projectButtonTooltip'>{{ ::'editConstraints' | translate }}</md-tooltip>
   </md-button>
-  <md-button ng-if='!$ctrl.isGroupNode($ctrl.node.id)'
+  <md-button ng-if='!$ctrl.node.isGroup()'
```

Comment:
Should the isGroup value be stored in a variable in the controller and then be used in the template so the template doesn't have to call the isGroup() function so many times. This would cut down on the number of times the template watcher needs to evaluate the ng-if since it would be a variable and not a function call.

Reply:
Good suggestion. I made the change.